infrastructure/module/eks3/main.tf

```diff
@@ -0,0 +1,65 @@
+module "eks" {
+  source  = "terraform-aws-modules/eks/aws"
+  version = "20.10.0"
+
+  cluster_name    = "igork-cluster"
+  cluster_version = "1.29"
+
+  cluster_endpoint_public_access  = true
+  cluster_endpoint_private_access = true
+
+  cluster_addons = {
```

Comment:
you are missing **aws-ebs-csi-driver**, you will need it for PVC once you provision PSQL in EKS.
It needs to have it's own irsa.